.github/workflows/ci.yml

```diff
@@ -23,6 +25,13 @@ jobs:
     - name: Build E2E Runner
       shell: cmd
       run: dotnet.exe build src\Ztm.EndToEndTests
+    - name: Install main database
+      shell: cmd
+      run: choco.exe install postgresql --version=10.6.0.20190131 --params '/Password:postgres' --params-global
+    - name: Migrate main database
+      shell: cmd
+      working-directory: ./src/Ztm.Data.Entity.Postgres
```

Comment:
Path separator in Windows should be `\`. Also I think we can omit `.` at the beginning.

Reply:
I'm not sure this field comply the Windows directory format and can't see the spec. However my Visual Studio Code show error message when I use Windows's format. This is the error message `String does not match the pattern of "^(\.[^\\?%*:|"<>\n]+)$".`

Reply:
Have you tried this? `src\Ztm.Data.Entity.Postgres`

Reply:
It show the same error